File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/comment/direct/Java/C/s063482767_commented.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    int sum = 0;
    int max = 0;
    int num;
    for (int i = 0; i < n; i++) {
        scanf("%d", &num);
        sum += num;
        if (num > max) {
            max = num;
        }
    }
    printf("%s\n", sum > 2 * max ? "Yes" : "No");
    return 0;
}
// End of Code
i < n - 1; i++) {
        sum += arr[i];
    }
    
    printf(arr[n - 1] < sum ? "Yes\n" : "No\n");
    
    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/comment/direct/Java/Go/s063482767_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strconv"
)

func main() {
	sc := bufio.NewScanner(os.Stdin)
	sc.Split(bufio.ScanWords)
	var tokens []string
	for sc.Scan() {
		tokens = append(tokens, sc.Text())
	}
	n, _ := strconv.Atoi(tokens[0])
	nums := make([]int, n)
	for i := 0; i < n; i++ {
		num, _ := strconv.Atoi(tokens[1+i])
		nums[i] = num
	}
	sort.Ints(nums)
	sum := 0
	for _, v := range nums[:len(nums)-1] {
		sum += v
	}
	if nums[len(nums)-1] < sum {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

// End of Code
